documentation/modules/auxiliary/scanner/http/icinga_static_library_file_directory_traversal.md

```diff
@@ -0,0 +1,74 @@
+
+## Vulnerable Application
+
+Icingaweb versions between 2.9.0-2.9.5, 2.8.0-2.8.5 (inclusive) suffer from an
+unauthenticated directory traversal vulnerability. The vulnerability is triggered
+through the icinga-php-thirdparty library, and the file to read is an absolute path.
```

Comment:
As which user? The icinga one?

Reply:
@h00die I think this does execute as the icinga user.  Any chance we could add that to the documentation?  I think this is ready to land after that addition, and I'm happy to add it myself if you'd like.

Reply:
as long as youre in there to merge it, may as well tack on a commit as well. thanks @bwatters-r7

Reply:
This will be updated as part of a few updates I'm doing to this module whilst reviewing it.

Reply:
Fixed with https://github.com/rapid7/metasploit-framework/pull/17915/commits/cf6b3099041bb04815263c0a62336ceb7e07f502